Cargo.toml

```diff
@@ -29,9 +30,10 @@ rustversion = "1.0.7"
 
 [features]
 default = ["pretty_errors", "module_disambiguation"]
-pretty_errors = ["proc-macro-error"]
+pretty_errors = ["dep:proc-macro2-diagnostics", "dep:proc-macro2"]
```

Comment:
What does `dep:` do here?

Reply:
`dep:` stops the optional dependency from becoming an implicit feature.

From the [Cargo Book](https://doc.rust-lang.org/cargo/reference/features.html#optional-dependencies):
> In some cases, you may not want to expose a feature that has the same name as the optional dependency. For example, perhaps the optional dependency is an internal detail, or you want to group multiple optional dependencies together, or you just want to use a better name. If you specify the optional dependency with the `dep:` prefix anywhere in the `[features]` table, that disables the implicit feature.

Reply:
Wow I didn't know about this. Thank you for using it and getting it to my attention. I think this probably needs a mention in the changelog